lib/expression/callbacks/standard.ex

```diff
@@ -856,13 +856,13 @@ defmodule Expression.Callbacks.Standard do
       stop > 0 ->
         binary
         |> String.split(@punctuation_pattern)
-        |> Enum.slice((start - 1)..(stop - 2))
+        |> Enum.slice((start - 1)..(stop - 2)//1)
```

Comment:
what does this do?

Reply:
It slices backwards starting at the end. This used to assume a step of 1 when the slice is negative, elixir 1.16 wants the step listed explicitly.